### PR TITLE
chroma: add page

### LINKED ISSUE #4515: page request: <chroma>, with added samples
### Chroma is a general purpose syntax highlighting library and corresponding command
Repo: https://github.com/alecthomas/chroma

### Samples
```sh
# colorize with shell style
chroma -l sh
# Fuzzy search colorized aliases
alias | chroma -l sh | fzf --ansi
```

### PR DESCRIPTION
**Pull request by Starkiller645**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Fixes #4515

### CODE REVIEW

**Review by owenvoke — CHANGES_REQUESTED** [review]
It looks like all of these are the same, so it should be fine to merge them into a single command under the `pages/common` directory. 👍🏻

**Comment by Starkiller645** [comment]
I've moved the file into the common directory :)

File: pages/common/chroma.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# chroma
+
+> Chroma is a general-purpose syntax highlighting library and corresponding command, for Go.
+> More information: <https://github.com/alecthomas/chroma>.
+
+- Highlight a source file with python lexer and output to terminal:
+
+`chroma --lexer='python' {{source_file}}`
```
Comment (navarroaxel) [inline pages/common/chroma.md:8]: The argument options should be between `{}`:
```suggestion
`chroma --lexer="{{python}}" {{source_file}}`
```

File: pages/common/chroma.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Highlight a source file with Go lexer and output to a HTML file:
+
+`chroma --lexer="go" --formatter="html" {{source_file}} > {{html_file}}`
```
Comment (navarroaxel) [inline pages/common/chroma.md:12]:
```suggestion
`chroma --lexer="{{go}}" --formatter="{{html}}" {{source_file}} > {{html_file}}`
```

File: pages/common/chroma.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Highlight a source file with C++ lexer and output to an SVG image, using the Monokai style:
+
+`chroma --lexer="c++" --formatter="svg" --syle="monokai" {{source_file}} > {{svg_file}}`
```
Comment (navarroaxel) [inline pages/common/chroma.md:16]:
```suggestion
`chroma --lexer="{{c++}}" --formatter="{{svg}}" --syle="{{monokai}}" {{source_file}} > {{svg_file}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @Starkiller645, welcome and thanks for your submission. I left some comments.

**Review by navarroaxel — APPROVED** [review]

**Review by owenvoke — APPROVED** [review]

**Pull request merged**